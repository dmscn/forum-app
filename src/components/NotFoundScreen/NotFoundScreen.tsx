import React from 'react';

import logo from '../../assets/svg/logo-mini.svg';
import styles from './NotFoundScreen.module.css';
import Text from '../UI/Text';

export default function NotFoundScreen() {
  return (
    <main className={styles.Container}>
      <img src={logo} />
      <Text variant={'dialog'}>Oops! Wrong path, pal...</Text>
    </main>
  );
}
