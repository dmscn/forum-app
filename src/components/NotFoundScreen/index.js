export { default } from './NotFoundScreen';
export * from './NotFoundScreen';
