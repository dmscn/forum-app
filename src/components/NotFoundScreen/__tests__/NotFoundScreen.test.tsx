import React from 'react';
import NotFoundScreen from '../NotFoundScreen';
import { shallow } from 'enzyme';

describe('<NotFoundScreen />', () => {
  it('renders', () => {
    const wrapper = shallow(<NotFoundScreen />);
    expect(wrapper).toMatchSnapshot();
  });
});
