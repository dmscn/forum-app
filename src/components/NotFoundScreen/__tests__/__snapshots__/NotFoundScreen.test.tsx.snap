// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundScreen /> renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFoundScreen />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          src="logo-mini.svg"
        />,
        <HeaderText>
          Wrong path, pal...
        </HeaderText>,
      ],
      "className": "Container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "src": "logo-mini.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Wrong path, pal...",
        },
        "ref": null,
        "rendered": "Wrong path, pal...",
        "type": [Function],
      },
    ],
    "type": "main",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            src="logo-mini.svg"
          />,
          <HeaderText>
            Wrong path, pal...
          </HeaderText>,
        ],
        "className": "Container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "src": "logo-mini.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Wrong path, pal...",
          },
          "ref": null,
          "rendered": "Wrong path, pal...",
          "type": [Function],
        },
      ],
      "type": "main",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
